### ISSUE TITLE
[Bug]: Matplotlib freezing after calling plot function in clean Anaconda install

### ISSUE THREAD

**Issue by timmocking** [matplotlib/matplotlib#21846]
### Bug summary

Somehow, my plots are no longer generating. Calling plots freezes either in Spyder, Notebooks or just from a regular terminal.

### Code for reproduction

```python
import matplotlib.pyplot as plt

plt.plot([1, 2, 3, 4])
```

### Actual outcome

When calling in Jupyter or Spyder, the kernel becomes unresponsive. When using "python3" or "python" in the terminal, it keeps running endlessly.

### Expected outcome

I would expect a plot, or at the very least an error message.

### Additional information

I thought this was an issue with ipython for a while, but the issue persists when using python 3.9.7 in the terminal.

I did a couple of fresh re-installs of Anaconda on my Ubuntu system, without success. 

### Operating system

Ubuntu

### Matplotlib Version

3.4.3

### Matplotlib Backend

Qt5Agg

### Python version

3.9.7

### Jupyter version

6.4.5

### Installation

conda

**Comment by tacaswell** [comment]
does

```python
import matplotlib
matplotlib.use('tkagg')
import matplotlib.pyplot as plt
plt.plot(range(5))
plt.show()
```

work?

**Comment by timmocking** [comment]
No, that doesn't work unfortunately. To add some extra details, I'm doing all this on a dual-boot system with Ubuntu and Windows both installed (if that matters, I don't think it does). Everything was working fine.

I also figured out that the issue is completely dependent on my conda base environment, which is why I did not stumble upon this issue until I started using an IDE.

The following environment solves the problem:

```
name: debug
channels:
  - conda-forge
  - defaults
dependencies:
  - python=3.9
  - pip
  - pip:
    - matplotlib
```
The python version is equal between the base and debug environment. So I've located the issue with the conda installation of matplotlib. Pip installs version 3.5.0 of matplotlib which seems to work fine.
